### PR TITLE
Implement detail view page

### PR DESCRIPTION
**Pull request by erikdvt**
**Detail view page:**

- Shows details about a single comic
- Has buy button which directs user to Marvel website in Safari to buy comic
- Add button - TODO
- Has auto-layout with scroll view for landscape
- Tab bars and navigation bar implemented as well

**Screenshots:**
Simulator Screen Shot - iPhone 11 - 2022-04-22 at 08 36 02
Simulator Screen Shot - iPhone 11 - 2022-04-22 at 08 36 13

### CODE REVIEW

File: Marv-Lous/Marv-Lous/Controllers/DetailsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        comicReleaseDate.text = comicData.releaseDate
+        if let thumbnailURL = comicData.thumbnailLink {
+            comicThumbnail.loadImageFromURL(imageURL: thumbnailURL)
+        }
+    }
+}
+
+extension DetailsViewController: DetailsViewModelDelegate {
+    func showError(_ error: String) {
+        print(error)
```
Comment (bgouws) [inline Marv-Lous/Marv-Lous/Controllers/DetailsViewController.swift]: Think we can handle this in this PR 😄

**Review by bgouws — CHANGES_REQUESTED** [review]
Very nice PR -lets just handle that error 😄

**Review by bgouws — APPROVED** [review]
Looks good 😄

**Review by KirstyPallatte — APPROVED** [review]
Looking good 👍🏻

**Pull request merged**